packages/kbn-esql-validation-autocomplete/src/autocomplete/__tests__/autocomplete.suggest.eval.test.ts

```diff
@@ -549,5 +549,104 @@ describe('autocomplete.suggest', () => {
         { triggerCharacter: ' ' }
       );
     });
+
+    test('case', async () => {
+      const { assertSuggestions } = await setup();
+      const comparisonOperators = ['==', '!=', '>', '<', '>=', '<=']
+        .map((op) => `${op} `)
+        .concat(',');
+
+      // case( / ) suggest any field/eval function in this position as first argument
+      await assertSuggestions(
+        'from a | eval case(/)',
+        [
+          // With extra space after field name to open suggestions
+          ...getFieldNamesByType('any').map((field) => `${field} `),
+          ...getFunctionSignaturesByReturnType('eval', 'any', { scalar: true }, undefined, [
+            'case',
+          ]),
+        ],
+        {
+          triggerCharacter: ' ',
+        }
+      );
```

Comment:
Let's test this without a trigger character as well.

Reply:
Added here [`9c6002b` (#192135)](https://github.com/elastic/kibana/pull/192135/commits/9c6002b5c959a9f88f54e40c7e516fd33e74bafb)